src/corenfc.cs

```diff
@@ -144,9 +152,141 @@ interface NFCIso15693Tag : NFCTag {
 		[Abstract]
 		[Export ("readMultipleBlocksWithConfiguration:completionHandler:")]
 		void ReadMultipleBlocks (NFCIso15693ReadMultipleBlocksConfiguration readConfiguration, Action<NSData, NSError> completionHandler);
+
+		[iOS (13,0)]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("stayQuietWithCompletionHandler:")]
+		void StayQuiet (Action<NSError> completionHandler);
```

Comment:
Do we need async here since it takes a handler?

Reply:
no, it's a protocol